### DISCUSSION TITLE
What is "Auto-refresh current timesheet"?

### DISCUSSION THREAD

**Discussion by JamesInform** [AlexandrePTJ/kemai#109] [Q&A] [answered]
Hi

I am just testing your tool. Looks great.

One question:
What is "Auto-refresh current timesheet"
![image](https://github.com/AlexandrePTJ/kemai/assets/38714600/248104f8-f2fb-4d63-a83b-1b116fad79ec)

Cheers,
James

**Comment by AlexandrePTJ** [comment, marked as answer]
It will check every x seconds status of the current timesheet. Useful when using kimai from several devices.
